openshift/client/apis/authorization_openshift_io_api.py

```diff
@@ -0,0 +1,138 @@
+
+    OpenAPI spec version: v3.6.0-alpha.0
+    
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+from __future__ import absolute_import
+
+import sys
+import os
+import re
+
+# python 2 and python 3 compatibility library
+from six import iteritems
+
+from kubernetes.client.configuration import Configuration
```

Comment:
Yes, since the kubernetes library provides the configuration parsing that we need we are using that rather than wrapping it or re-implementing.